docs/upgrade/2.10.1_to_2.11.0.rst

```diff
@@ -0,0 +1,93 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.10.1 to 2.11.0
+=============================
+
+Preparing for the Ubuntu 24.04 (Noble) migration
+------------------------------------------------
+
+The 2.11.0 release includes a number of features that will help ensure
+your SecureDrop server is prepared for the automated migration to Ubuntu 24.04
+(Noble) in early 2025.
+
+After updating to SecureDrop 2.11.0 using the steps below, it's important
+that you review our :doc:`Ubuntu 24.04 (Noble) migration guide <../admin/maintenance/noble_migration_prep>`,
+and run the ``securedrop-noble-migration-check`` script prior to January 31st, 2025
+to ensure your server is ready for the migration.
```

Comment:
Do you want to add something like, "We will have more details on the migration itself early next year"?